### PR TITLE
feat: add new component Pagination

### PR DESCRIPTION
**Pull request by jhoncool**
Added new component Pagination

### CODE REVIEW

File: src/components/Pagination/components/PaginationInput/PaginationInput.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    const handleBlur: TextInputProps['onBlur'] = (event) => handleUpdate(event.currentTarget.value);
+
+    const handleKeyUp: TextInputProps['onKeyUp'] = (event) => {
+        if (event.key === KeyCode.ENTER) {
+            handleUpdate(event.currentTarget.value);
+        }
+    };
+
+    return (
+        <TextInput
```
Comment (korvin89) [inline src/components/Pagination/components/PaginationInput/PaginationInput.tsx:54]: Maybe use input with `type="number"`?

Comment (amje) [reply src/components/Pagination/components/PaginationInput/PaginationInput.tsx:54]: Better check input by regex `[1-9][0-9]*`, `type="number"` also allows `1e3`-ish values

Comment (jhoncool) [reply src/components/Pagination/components/PaginationInput/PaginationInput.tsx:55]: fixed

**Review by korvin89 — APPROVED** [review]

File: src/components/Pagination/components/PaginationButton/PaginationButton.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            button = (
+                <Button
+                    size={size}
+                    view="outlined"
+                    className={className}
+                    onClick={() => onUpdate(page - 1, pageSize)}
+                    title={compact ? i18n('button_previous') : undefined}
+                    disabled={disabled}
+                >
+                    <Icon data={ChevronLeft} size="14" />
```
Comment (amje) [inline src/components/Pagination/components/PaginationButton/PaginationButton.tsx:58]: It should be 16 for m button size I think

File: src/components/Pagination/components/PaginationInput/PaginationInput.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            className={b({size}, className)}
+            placeholder={i18n('label_input-placeholder')}
+            size={size}
+            value={value}
+            onUpdate={setValue}
+            onBlur={handleBlur}
+            onKeyUp={handleKeyUp}
+            type="number"
+            controlProps={{pattern: '[1-9][0-9]*'}}
```
Comment (amje) [inline src/components/Pagination/components/PaginationInput/PaginationInput.tsx:64]: Not working now, I can still write `1e-2`

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Pagination/components/PaginationButton/PaginationButton.tsx (hunk 2 above)
Comment (jhoncool) [reply src/components/Pagination/components/PaginationButton/PaginationButton.tsx]: fixed

File: src/components/Pagination/components/PaginationInput/PaginationInput.tsx (hunk 3 above)
Comment (jhoncool) [reply src/components/Pagination/components/PaginationInput/PaginationInput.tsx]: fixed

**Review by amje — APPROVED** [review]

**Pull request merged**